deps.edn

```diff
@@ -37,11 +36,13 @@
                                           :sha "1161efe7f5fd5abb066adf2283f5da3c7683a48f"}
 
          ;; cryptography
-         com.fluree/crypto               {:mvn/version "0.3.9"}
+         com.fluree/crypto               { ;; :mvn/version "0.3.9"
+                                          :git/url "https://github.com/fluree/fluree.crypto.git"
+                                          :sha "3eaf1bdafa4395f58d2e5105cfd77fb341126fe1"}
          org.bouncycastle/bcprov-jdk15on {:mvn/version "1.70"}
 
          ;; smartfunctions
-         org.babashka/sci                {:mvn/version "0.3.32"}}
+         org.babashka/sci                {:mvn/version "0.3.31"}}
```

Comment:
Why the downgrade?

Reply:
There's a weird interaction between shadow-cljs and the specific version of clojurescript we were using that causes sci to throw a null pointer exception when loaded. I'm working with Borkdude on finding a resolution but for now this gets everything to compile.